lib/server/services/qualityControlFlag/QcFlagService.js

```diff
@@ -546,6 +550,32 @@ class QcFlagService {
         };
     }
 
+    /**
+     * Find QC flags in GAQ effective periods for given data pass and run
+     *
+     * @param {number} dataPassId id od data pass
+     * @param {number} runNumber run number
+     * @return {Promise<GaqQcFlags>} promise of aggregated QC flags
+     */
+    async getGaqQcFlags(dataPassId, runNumber) {
+        const effectiveGaqPeriods = await QcFlagRepository.findFlagsInGaqEffectivePeriods({ dataPassId, runNumbers: [runNumber] });
```

Comment:
why is the variable named `effectiveGaqPeriods` ?

Reply:
because it, effective periods (but different than QC flag effective period) in which there is list of QC flags contributing to GAQ, but I renamed it

Reply:
changed to `GaqPeriods`